feature-libs/order/components/order-details/order-detail-actions/order-detail-actions.component.html

```diff
@@ -10,7 +10,7 @@
     </div>
 
     <div class="col-xs-12 col-md-4 col-lg-3">
-      <a
+      <button
```

Comment:
Changes to the DOM structure even if minor are still consider breaking. We will need a feature flag here. I've left some tips how to deal with them on the previous PR. 🙂

Reply:
I've just updated it.